model/qti/ImportService.php

```diff
@@ -84,6 +82,12 @@ class ImportService extends ConfigurableService
      */
     const CONFIG_VALIDATE_RESPONSE_PROCESSING = 'validateResponseProcessing';
 
+    /**
+     * TTL of the item importing process
+     * How long item will be locked while lock service automatically release the lock
+     */
+    const OPTION_IMORT_LOCK_TTL = 'importLockTtl';
```

Comment:
A typo, I guess it should be `OPTION_IMPORT_LOCK_TTL`

Reply:
Good catch! thank you